Misc/NEWS.d/next/Library/2020-09-08-23-41-29.bpo-41747.M6wLKv.rst

```diff
@@ -0,0 +1,3 @@
+Ensure all methods belonging to :func:`dataclasses.dataclass` objects
+now have the proper ``__qualname__`` attribute referring to the class
+they belong. Patch by Batuhan Taskaya.
```

Comment:
Again, I think this sentence should end in "they belong to" (adding "to"). It reads better that way.